recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 65364d6eb1da26fd7aba61770c4deef4a7d6f1220799c362750a3f1ac89c8f89
 
 build:
-  number: 0
+  number: 1000
```

Comment:
@tadeu and @nicoddemus you probably want to remove the skip below.

Reply:
Yep, thanks for the ping, @tadeu is on it! 😉